pydis_site/apps/api/views.py

```diff
@@ -37,12 +37,14 @@ class RulesView(APIView):
 
     ## Routes
     ### GET /rules
-    Returns a JSON array containing the server's rules:
+    Returns a JSON array containing the server's rules
+    and keywords relating to each rule.
+    Example response:
 
     >>> [
-    ...     "Eat candy.",
-    ...     "Wake up at 4 AM.",
-    ...     "Take your medicine."
+    ...     ("Eat candy.", ["candy", "sweets"]),
+    ...     ("Wake up at 4 AM.", ["wake_up", "early", "early_bird"]),
+    ...     ("Take your medicine.", ["medicine", "health"])
```

Comment:
Representing the data as tuples in docs is potentially misleading because JSON does not distinguish between tuples and lists once serialised.

Reply:
You're correct, thank you !